style.css

```diff
@@ -0,0 +1,61 @@
+.video-section {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax
+  (250px, 1fr));
+  gap: 3rem 1rem;
+  padding: 3rem 0;
+  margin: 0 1.5rem;
+  border-top: 4px solid #ccc;
+}
+
+.video-section:first-child {
+  border-top: none;
+}
+
+.video-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.video-bottom-section {
+  display: flex;
+  align-items: flex-start;
+  margin-top: 1rem;
+}
+
+
+.channel-icon {
+  margin-right: .75rem;
+  border-radius: 50%;
+  width: 36px;
+  height: 36px;
+  background-color: #aaa;
+}
+
+.video-details {
+  display: flex;
+  flex-direction: column;
+}
+
+.video-title {
+  font-size: 1.1rem;
+  font-weight: bold;
+  margin-bottom: 5rem;
+  text-decoration: none;
+  color: black;
+}
+
+.video-channel-name {
+  margin-bottom: 1rem;
+  text-decoration: none;
+  color: #555
+}
+
+.video-channel-name:hover {
+  color: #111;
+}
+
+.video-channel-name,
+.video-metadata {
+ color: #555;
+}
```

Comment:
I love the way your CSS looks, its got a great layout 💯 

- However, just work on the spacing on your styling to avoid linter errors 👍

Reply:
- [ ] I think it would be better to consider using more polite comments and explain clearly the error so it can be for your easy peer to understand.  Please try to be specific on which part can cause linter errors and your suggestion on how it can e solved.